.github/workflows/notebook-controller-images-updater.yaml

```diff
@@ -83,14 +112,14 @@ jobs:
           git add components/notebook-controller/config/overlays/openshift/params.env
           git add components/odh-notebook-controller/config/base/params.env
           git add components/odh-notebook-controller/makefile-vars.mk
-          git commit -m ":robot: Update odh and notebook-controller with image ${VERSION}-${COMMIT_ID}"
+          git commit -m "Update odh and notebook-controller with image ${VERSION}-${GIT_HASH}"
           git push origin ${{ env.TEMP_UPDATER_BRANCH }}
-          git log --oneline
         else
           echo "There were no changes detected on ${{ env.BRANCH_NAME }}"
         fi
-        
-    - name: Create Pull Request
+
+    - name: PR Cretation
+      if: ${{  env.GENERATE_PR == 'true' }}
       run: |
           gh pr create --repo https://github.com/$REPO_OWNER/$REPO_NAME.git \
```

Comment:
I don't think you need to provide the hostname when using `gh` to the `--repo` argument